srv/_modules/lyveutils.py

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+# How to test:
+# salt-call saltutil.clear_cache && salt-call saltutil.sync_modules
+# salt-call lyveutils.decrypt "component" "secret"
+
+import logging
+import os
+import sys
+
+# Update PYTHONPATH to include Provisioner API installed at:
+# /usr/local/lib/python3.6/site-packages/cortx-prvsnr-*
+sys.path.append(os.path.join(
```

Comment:
that logic is not good since it is based on hard-coded assumption regrading provisioner installation path and finally it's a copy-paste. How to improve:
- move that into some shared location (it is what I referred in my previous review) and re-use it from there for any module
- use [this](https://github.com/Seagate/cortx-prvsnr/blob/7ab6832cc2add72c9c2d386e185933f50f1a0d6e/srv/_modules/prvsnr.py#L27-L50) logic that does similar but without any hard-codes